mobile.css

```diff
@@ -1678,4 +1678,688 @@ button:active {
+  }
+
+  .images img {
+    width: 184px;
+  }
+
+  .eclipse1,
+  .eclipse2 {
+    display: none;
+  }
+
+  .content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 114px 0;
+    position: absolute;
+    width: 1156px;
+    height: 1121px;
+    left: 100px;
+    top: 72px;
+  }
+
+  .big-circle1 {
+    position: absolute;
+    width: 184px;
+    height: 184px;
+    left: 0;
+    top: 0;
+    display: flex;
+  }
+
+  .big-circle2 {
+    position: absolute;
+    width: 184px;
+    height: 184px;
+    right: 0;
+    bottom: 0;
+    display: flex;
+  }
+
+  .item-links {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    position: absolute;
+    width: 241px;
+    height: 40px;
+    right: 164px;
+    top: 6px;
+    list-style-type: none;
+    justify-content: center;
+  }
+
+  .item1 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 10px 12px;
+    position: static;
+    width: 77px;
+    height: 40px;
+    left: 164px;
+    top: 0;
+  }
+
+  .item2 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 10px 12px;
+    position: static;
+    width: 65px;
+    height: 40px;
+    left: 91px;
+    top: 0;
+  }
+
+  .item3 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 10px 12px;
+    position: static;
+    width: 77px;
+    height: 40px;
+    left: 164px;
+    top: 0;
+  }
+
+  .item-links a {
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 15px;
+    line-height: 20px;
+    text-decoration: none;
+    color: #fff;
+  }
+
+  .logo {
+    position: static;
+    width: 77px;
+    height: 20px;
+    left: 12px;
+    top: 10px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+  }
+
+  .email-icon {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    padding: 8px;
+    position: absolute;
+    width: 40px;
+    height: 40px;
+    right: 112px;
+    top: calc(50% - 40px / 2);
+  }
+
+  .name-header {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0;
+    position: static;
+    width: 1156px;
+    height: 192px;
+    left: calc(50% - 1156px / 2);
+    top: 114px;
+  }
+
+  .primary-text {
+    position: static;
+    width: 1156px;
+    height: 96px;
+    left: 0;
+    top: 0;
+  }
+
+  .primary-text h1 {
+    text-transform: uppercase;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 60px;
+    line-height: 105px;
+    text-align: center;
+  }
+
+  .support-text {
+    position: static;
+    width: 1156px;
+    height: 72px;
+    left: 0;
+    top: 120px;
+  }
+
+  .support-text h3 {
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 40px;
+    line-height: 47px;
+    text-align: center;
+  }
+
+  .content2 {
```

Comment:
Please use the same color for the paragraph test as can be seen in the design. Kindly see the screenshot below for reference.